routes/v1/utils.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const router = express.Router();
+
+const auth = require('middlewares/auth.js');
+
+const constants = require('utils/const.js');
+const safeHandler = require('utils/safeHandler');
+
+const {SocksProxyAgent} = require('socks-proxy-agent');
+const axios = require('axios');
+
+const agent = new SocksProxyAgent(`socks5h://${constants.TOR_PROXY_IP}:${constants.TOR_PROXY_PORT}`);
```

Comment:
What does the `h` mean in `socks5h` here?

Does it still work ok if you remove it?

Reply:
Based on [this part](https://github.com/TooTallNate/node-socks-proxy-agent/blob/8ed8efa99c8a17a8ee52872728f13a94497ddc13/src/agent.ts#L51) of package source, the `5h` corresponds to the socks version (as it's possible to use other versions like `4`, `4a`, or `5`). If no version is specified, the default is set to `5h`. So deleting it won't do anything, but the socks version is clear. As you like.

Reply:
According to [this](https://github.com/urllib3/urllib3/issues/1035#issue-188967574) `socks5h` means the hostname is resolved by the SOCKS server and `socks5` means it's resolved locally.

So yeah, we definitely want `socks5h`!